cypress/e2e/backoffice/product-measurement-unit/product-measurement-unit-create.cy.ts

```diff
@@ -0,0 +1,56 @@
+import { container } from '@utils';
+import { UserLoginScenario } from '@scenarios/backoffice';
+import {
+  ProductMeasurementUnitCreatePage
+} from '../../../support/pages/backoffice/product-measurement-unit/create/product-measurement-unit-create-page';
+import {
+  ProductMeasurementUnitListPage
+} from '../../../support/pages/backoffice/product-measurement-unit/list/product-measurement-unit-list-page';
+import {
+  ProductMeasurementUnitManagementStaticFixtures
+} from '../../../support/types/backoffice/product-measurement-unit-management';
+
+describe('Measurement Units - Create', { tags: ['@backoffice', '@product-measurement-unit'] }, () => {
```

Comment:
Please note that if you release this new feature in the suite-nonsplit along with these tests before it’s been integrated into the demoshops, those tests will very likely fail in the demoshops environment.

Right now there are 2 ways on how to deal with it:
- release -> integrate -> merge tests (less preferred as you can't 'test' your integration in demoshops with these autotests)
- skip these tests for all demoshops (repositories) until you integrate this new feature in demoshops (and unskip tests once integration is done), like here https://github.com/spryker/cypress-tests/blob/master/cypress/e2e/yves/multi-factor-authentication/customer-mfa-auth.cy.ts#L16